### PR TITLE
Add pages "troubleshooting" and "other installation methods" 

### PR DESCRIPTION
**Pull request by Nutomic**
I'm open for any suggestions to improve the text. Also tell me if you can think of other problems to add in the troubleshooting page.

@realkinetix @revmischa If you write a better description text for your projects, you can comment here and I will insert it, or make a PR yourself after this one is merged.

### CODE REVIEW

**Review by revmischa — APPROVED** [review]
It looks good to me!

File: src/en/administration/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+proxy_set_header X-Real-IP $remote_addr;
+proxy_set_header Host $host;
+proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+```
+
+## Federation
+
+### Other instances can't fetch local objects (community, post, etc)
+
+Your reverse proxy (eg nginx) needs to forward requests with header `Accept: application/activity+json` to the backend. This is handled by the following lines:
```
Comment (dessalines) [inline src/en/administration/troubleshooting.md:37]: Also link to the [nginx template](https://github.com/LemmyNet/lemmy/blob/main/ansible/templates/nginx.conf) somewhere here.

File: src/en/administration/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+proxy_pass $proxpass;
+```
+
+You can test that it works correctly by running the following commands, all of them should return valid JSON:
+```
+curl -H "Accept: application/activity+json" https://your-instance.com/u/some-local-user
+curl -H "Accept: application/activity+json" https://your-instance.com/c/some-local-community
+curl -H "Accept: application/activity+json" https://your-instance.com/post/123 # the id of a local post
+curl -H "Accept: application/activity+json" https://your-instance.com/comment/123 # the id of a local comment
```
Comment (dessalines) [inline src/en/administration/troubleshooting.md:54]: Thanks for these, this is helpful to have.

**Review by dessalines — CHANGES_REQUESTED** [review]
Thx for making!

File: src/en/administration/troubleshooting.md (hunk 1 above)
Comment (Nutomic) [reply src/en/administration/troubleshooting.md:39]: Okay, added.

**Pull request merged**